### PR TITLE
Homepage cms

### PR DESCRIPTION
**Pull request by jomarmontuya**
# Description

Connect the homepage back to CMS

Fixes # (issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list the simulator and android emulators used.

- [x] Manual Test

### CODE REVIEW

**Comment by jomarmontuya** [comment]
@ardeay  This is ready, tested on both, prod and stage environment

**Comment by jomarmontuya** [comment]
Sync and Tested with next13

File: src/views/zesty/Homepage.js [hunk 1]
```diff
@@ -90,17 +95,110 @@ function Homepage({ content }) {
...
+  const securityFeatureProps = {
+    overline: content?.security_feature_overline,
+    heading: content?.security_feature_heding,
+    supportingText: content?.security_feature_supporting_text,
+    image: content?.security_feature_image?.data?.[0]?.url,
+  };
+
   return (
     <>
       <ThemeProvider theme={() => revampTheme(palette.mode)}>
```
Comment (darwin808) [inline src/views/zesty/Homepage.js:191]: 2 themeproviders ?

File: src/revamp/ui/HeroV2/index.js [hunk 2]
```diff
@@ -267,13 +265,7 @@ const Hero = ({
...
         })}
       >
-        <img
-          src={media}
-          alt={generateAlt('Zesty image')}
-          loading="eager"
-          width={'100%'}
-          height={'100%'}
-        />
+        <img src={heroImage} alt={generateAlt('Zesty image')} loading="eager" />
```
Comment (darwin808) [inline src/revamp/ui/HeroV2/index.js:268]: widht and ht?

**Review by giseleblair — APPROVED** [review]

File: src/views/zesty/Homepage.js [hunk 3]
```diff
@@ -74,6 +74,11 @@ const GetDemoSection = dynamic(() => import('revamp/ui/GetDemoSection'), {
   loading: () => <p>Loading...</p>,
 });
 
+export const removeHTMLtags = (str) => {
```
Comment (darwin808) [inline src/views/zesty/Homepage.js:77]: move this in utils file we can use it later

**Pull request merged**